cocos/serialization/deserialize.ts

```diff
@@ -1022,7 +1110,22 @@ export function unpackJSONs (
     const sections = data[PACKED_SECTIONS];
     for (let i = 0; i < sections.length; ++i) {
         const section = sections[i];
+        const binaryStorageSpan = section[PACKED_FILE_SECTION_BINARY_STORAGE_INDEX];
         (section as any[]).unshift(version, sharedUuids, sharedStrings, sharedClasses, sharedMasks);
+        if (binaryStorageSpan !== EMPTY_PLACEHOLDER) {
+            if (!binaryChunk) {
+                // Bad data: there's section requiring binary storage but the incoming data didn't provide one. 
+                throw new Error(`Bad data: there's section requiring binary storage but the incoming data didn't provide one`);
+            }
+
+            const [byteOffset, byteLength] = binaryStorageSpan;
+            const span = new Uint8Array(
```

Comment:
Can't we provide only one sharedBuffer, like sharedMasks?

Reply:
As discussion with @holycanvas , we agree that here we need to copy the section memory(ac1152581585ab6b69b4e1eae73ecfaee14bf986). So we can't do sharing.